string Class
float64 UTM_x
float64 UTM_y
float64 Height
